src/style.css

```diff
@@ -1 +1,29 @@
 /* add styles here */
+.body {
+  margin: 0;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.classes`. If you intended to style the `<body>` element, the correct selector is `body` without the dot. If `.body` is a class, consider renaming it to represent the content or component it styles, rather than the tag name.